src/lev-report/charts/HourlyUsage.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { VictoryAxis, VictoryChart, VictoryLabel, VictoryLine, VictoryTheme } from 'victory';
+
+const hours = Array.from({ length: 25 }, (_, h) => h);
```

Comment:
If this is the best ES6 can do then I guess we are stuck with things like this.

I would like to take the time to point out that in Haskell you can do things like:
```hs
hours = [1..24]
```
;-P

Reply:
Yeh the code golf for JS array creation and filling is pretty poor!
How's the Haskel support for React? Maybe we should re-code everything in Haskel.

Reply:
There's no React but I think there's a framework called Miso that let's you do component based stuff.

The problem is that setting up a good dev environment for compiling Haskell to JS is like pulling teeth while stabbing yourself with a fork. And also the HO doesn't have resource to support that sort of thing long-term.